README.md

```diff
@@ -1,8 +1,5 @@
-<p align="center">
-	<img src="./docs/images/Qbeast-spark.png" alt="Qbeast spark project"/>
-</p>
+![Qbeast-spark](./docs/images/Qbeast-spark.png)
```

Comment:
Is the image still centered?

Reply:
Ok the images should be centered again

Reply:
When I use  this code to center an image:
```
<p align="center">
![Qbeast-spark](./docs/images/Qbeast-spark.png)
</p>
```

The image looks like this in the website: 
<img width="874" alt="Screenshot 2024-07-31 at 16 39 41" src="https://github.com/user-attachments/assets/c8548504-17ad-4b74-9ce1-97990f6600da">

But looks like this in the qbeast-spark repository:
<img width="894" alt="Screenshot 2024-07-31 at 16 40 21" src="https://github.com/user-attachments/assets/7e881550-8515-4e00-a6b8-f862918240da">

Reply:
If I use the original code from the qbeast-spark repository:

```
<p align="center">
	<img src="./docs/images/Qbeast-spark.png" alt="Qbeast spark project"/>
</p>
```

The spark logo will show up in the github repository, but will look like this in the website:

<img width="865" alt="Screenshot 2024-07-31 at 16 56 41" src="https://github.com/user-attachments/assets/9aebdd86-f880-4664-99af-6a1f15a23039">